legacyLoginFragment/src/main/java/com/hamdan/forzenbook/legacy/login/fragment/LoginFragment.kt

```diff
@@ -78,7 +78,9 @@ class LoginFragment : Fragment() {
                 loginModel.state.collect { state ->
                     withContext(Dispatchers.Main) {
                         state.apply {
-
+                            if (this.loggedIn) {
+                                loginModel.login()
```

Comment:
return after this? you wouldn't want to keep going in this function right?

Reply:
Yea thats fair.